### DISCUSSION TITLE
Adding a checkbox to a table

### DISCUSSION THREAD

**Discussion by vennizemi** [sqlpage/SQLPage#286] [Q&A] [answered]
Hello everyone!

Please, help me.
I need to add a checkbox as a table column (with selectability).
Is there any way to do this?

example below

![image](https://github.com/lovasoa/SQLpage/assets/162549698/aa1a7335-7acf-48cd-942b-67cda0d31f98)

**Comment by lovasoa** [comment, marked as answer]
Hello! You can't do that with the built-in table component.

You can use the *form* component, optionally with a multiselect in it, or create your own custom component. 

https://sql.ophir.dev/documentation.sql?component=form#component

https://sql.ophir.dev/custom_components.sql

**Reply by vennizemi** [reply]
Thank yoy very much!
